src/Core/Services/Implementations/PolicyService.cs

```diff
@@ -57,10 +60,24 @@ public class PolicyService : IPolicyService
                         {
                             throw new BadRequestException("Single Sign-On Authentication policy is enabled.");
                         }
+
+                        var vaultTimeout =
```

Comment:
Can we move this into its own PR? This one is big enough already.

Reply:
Moved to https://github.com/bitwarden/server/pull/1694